src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(people => {
+        setPeopleFromServer(people);
+        setIsError(false);
+      })
+      .catch(() => setIsError(true))
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+      {!isError && (
+        <>
+          {peopleFromServer.length > 0 ? (
+            <PeopleTable people={peopleFromServer} isLoading={isLoading} />
```

Comment:
isLoading is not working now:
![image](https://github.com/mate-academy/react_people-table-basics/assets/86923089/46ddbd4d-d057-4284-9325-0d12ce8d72fa)

because `peopleFromServer` length will be 0 at this moment
handle it in this way (schematically):
```
 isLoading ? <Loader> : <PeopleTable people={peopleFromServer} />
 ```
 and check people.length inside the table and show `noPeopleMessage` if it's needed.